### PR TITLE
Update doc due to RTS community edition EOL

### PR DESCRIPTION
**Pull request by trusli**

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -6,32 +6,14 @@
 
 DataTorrent products are available for download from [https://www.datatorrent.com/download/](https://www.datatorrent.com/download)
 
--  **Community Edition**:  It is a packaged version of Apache Apex and enables developers to quickly develop their big data streaming and batch projects.
--  **Enterprise Edition**:  Designed for enterprise production deployment and includes security, advanced monitoring and troubleshooting, graphical application assembly, and application data visualization.
--  **Sandbox Edition**:  Enterprise Edition and demo applications pre-installed and configured with a single-node Hadoop cluster running in a virtual machine.  Optimized for evaluation and training purposes.
+-  **Sandbox Appliance**:  Quickly build real-time stream and batch data analytics applications with an all-in-one package delivered as an Oracle Virtualbox Appliance
```
Comment (sashadt) [inline docs/troubleshooting.md]: Appliance: (noun) "a device or piece of equipment designed to perform a specific task, typically a domestic one".  I suggest we stay with "Edition" or simply call it "Sandbox".

Virtualbox needs to be changed to VirtualBox.  

Message needs to explain that it comes pre-packaged with Apache Hadoop single node cluster.  And the "quick" part about it is user not having to install and configure this cluster.

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -6,32 +6,14 @@
...
+-  **Installable Binary**:  All packages (Apache Apex Core and RTS) include a license to our Free Edition that lets you explore all the capabilities of DataTorrent RTS with up to 128GB of processing capacity and community email support. Commercial and time-limited evaluation licenses are available if you need more processing capacity to validate your use case. Please note: Community Edition is now replaced by Free Edition.
```
Comment (sashadt) [inline docs/troubleshooting.md]: "128GB of processing capacity" is not clear.  Should refer to 128GB of memory.

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -201,7 +183,7 @@ You can contact us at [https://www.datatorrent.com/contact](https://www.datatorr
 
 # Installation
 
-There are multiple installations available e.g. Sandbox Edition, Community Edition, Enterprise Edition, dtIngest. Supported operating systems are which support Hadoop platform (tested on CentOS 6.x and Ubuntu 12.04).
+There are multiple installations available e.g. Sandbox Edition, Free/Enterprise Edition. Supported operating systems are which support Hadoop platform (tested on CentOS 6.x and Ubuntu 12.04).
```
Comment (sashadt) [inline docs/troubleshooting.md]: References "Sandbox Edition" but previous changed it to "Sandbox Appliance".  Should be ok once we remove "Appliance" and possibly go back to "Edition".

**Comment by lizygeogy** [comment]
@sashadt  The changes for editions were done based on SPOI-12737 and SPOI-12737. 
Please verify  to close this PR.

**Pull request closed without merging**